### ISSUE TITLE
Best practice for style library

### ISSUE THREAD

**Issue by MadalinNitu** [thaind97git/react-typescript-webpack#7]
What is the best practice to use a library like boostrap or material, or maybe a custom one in project. To import the same sass in all components

**Comment by thaind97git** [comment]
Yeah, this is a feature that I will implement and add to docs in the future. Thanks for your suggestion. I need to have more time to test other UI libraries.
